guides/deployment/deployment.md

```diff
@@ -14,6 +14,29 @@ As an example of deploying to other infrastructures, we also discuss four differ
 
 Let's explore those steps above one by one.
 
+#### Stickiness, Clustering, and the Long-Polling Transport {: .warning}
```

Comment:
In other for `{: .warning}` to work, you need to have everything within blockquotes, but this section is likely too big for that.  What if we move this to after "## Putting it all together" as its major section, called "Clustering and Long-Polling transports" and then we also preface by saying "If you are running more than one machine in production, which is the commend approach in most cases, ...".

Thank you for the well-written contribution.